website/addons/base/views.py

```diff
@@ -472,6 +494,16 @@ def addon_deleted_file(auth, node, **kwargs):
     """
     # Allow file_node to be passed in so other views can delegate to this one
     trashed = kwargs.get('file_node') or TrashedFileNode.load(kwargs.get('trashed_id'))
+    if kwargs.get('path'):
+        file_path = kwargs.get('path')
+    else:
+        file_path = trashed.path
+    if trashed.name:
+        file_name = trashed.name
+    else:
+        file_name = os.path.basename(file_path)
```

Comment:
Works great for file_path. Unfortunately, trashed.name is set to None, so doesn't hit default.